test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/assemble

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copy secrets into a location they can be output during image run
+mkdir -p $HOME/testsecret
+[[ -f /tmp/secret1 ]] && cp /tmp/secret? $HOME/testsecret 
```

Comment:
Also, if your test is specifically for `/tmp/secret`, there's no need for the wildcard.